src/Data/Swagger/Internal/Schema/Validation.hs

```diff
@@ -33,8 +34,9 @@ import           Data.HashMap.Strict                 (HashMap)
 import qualified Data.HashMap.Strict                 as HashMap
 import qualified Data.HashMap.Strict.InsOrd          as InsOrdHashMap
 import qualified "unordered-containers" Data.HashSet as HashSet
-import           Data.List                           (intercalate)
+#if __GLASGOW_HASKELL__ < 840
```

Comment:
we depend on ` base-compat-batteries`, so

```haskell
import Prelude ()
import Prelude.Compat
```

and remeoving `import Data.Monoid` will solve this without CPP

Reply:
:+1: 69b6c62be290edb05e87ff273c06f27bca33f975

thanks!